content/en/tools/starter-kits.md

```diff
@@ -23,6 +23,7 @@ Know of a Hugo-related starter kit that isn't mentioned here? [Please add it to
 The following starter kits are developed by active members of the Hugo community. If you find yourself having issues with any of the projects, it's best to file an issue directly with the project's maintainer(s).
 {{% /note %}}
 
+* [Wowchemy][]. Wowchemy is the **5,000+** ⭐️ open source Hugo starter kit and website builder **trusted by 500,000+ sites** since 2016. Create _any_ kind of site with [**50+ light/dark templates, widgets, and languages**](https://wowchemy.com/). Backed by a large, active community of **150+ contributors** :heart:
```

Comment:
This is a little effusive. Notice that the others do not have all the bold, emoji brilliance. Can't we tone it down a bit?

Reply:
@philoserf thank you for the feedback. We've removed the "bold, emoji brilliance" to focus just on the factual description of the open source project - please can you kindly review and merge if you're happy now? Thanks!

Reply:
Coolio. I am not a maintainer here. I would have one other question/comment. That has to do with ordering in the list. Why should _Wowchemy_ have the preeminent position at the head of the list? Hugo Wrapper and Victor Hugo have been here for a while. Even alphabetically Wow... would come later. Why this position?